website/content/en/docs/reference/metrics.md

```diff
@@ -146,8 +274,18 @@ Amount of time an interruption message is on the queue before it is processed by
 Count of messages deleted from the SQS queue.
 - Stability Level: STABLE
 
+## Cluster Metrics
+
+### `karpenter_cluster_utilization_percent`
+Utilization of allocatable resources by pod requests
+- Stability Level: ALPHA
+
 ## Cluster State Metrics
 
+### `karpenter_cluster_state_unsynced_time_seconds`
```

Comment:
Isn't this a new metric? Why put it at stable and not BETA?

Reply:
Yeah, this should be beta. Updated.